rest_registration/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2015-2019 Andrzej Pragacz
+# This file is distributed under the same license as
+# the Django-REST-Registration package.
+# Richard Savard <rax396@gmail.com>, 2019.
```

Comment:
I think it would be worth to either update this one or remove it entirely as it is not the true author of the translation.